modules/nf-core/freebayes/main.nf

```diff
@@ -9,11 +9,11 @@ process FREEBAYES {
 
     input:
     tuple val(meta), path(input_1), path(input_1_index), path(input_2), path(input_2_index), path(target_bed)
```

Comment:
```suggestion
    tuple val(meta1), path(input_1), path(input_1_index), path(input_2), path(input_2_index), path(target_bed)
```
to make it the same like the rest. anyway you can avoid just numbering them, by giving more insightful names?

Reply:
added the suggestion. 
Regarding better names for the metas: I have tried giving information on the meta.yml, but I worry that naming them things like `input_meta, reference_meta`, `samples_meta`, `populations_meta`, `cnv_meta` might be equally confusing. Specifically since "sample" and "population" might be information typically associated with the input channel in many uses 😬

Reply:
actually reverted that. I think it makes sense to keep the first meta as just meta and name the rest, for consistency with modules that only have one meta.
Otherwise, when using this module in a pipeline the dev will need to supply things like `prefix=${meta1.id}` in the `modules.config`, while for most other modules (incl. single meta modules) it would be `prefix=${meta.id}`. 
@mashehu Thoughts? Happy add it back.

Reply:
i would then give others clearer name like you said above. makes it also easier to rememeber which meta belongs to which input file.

Reply:
Done!